test/viewmodels/CashAccountV2ViewModelSpec.scala

```diff
@@ -55,7 +55,7 @@ class CashAccountV2ViewModelSpec extends SpecBase {
         shouldProduceCorrectAccountBalance(cashAccountViewModel.cashAccountBalance, eoriNumber, cashAccount)
         shouldProduceCorrectDownloadCSVFileLink(cashAccountViewModel.downloadCSVFileLinkUrl)
         shouldOutputCorrectHelpAndSupportGuidance(cashAccountViewModel.helpAndSupportGuidance)
-        shouldContainCorrectDailyStatementsSection(app, cashAccountViewModel.dailyStatementsSection.get, cashTransactions)
+        shouldContainCorrectDailyStatementsSection(application, cashAccountViewModel.dailyStatementsSection.get, cashTransactions)
```

Comment:
Line length is beyond 120, pls break it to next line. Same for Line 86

Reply:
Done